src/ActionModal/components/UndelegationModal.vue

```diff
@@ -364,7 +391,7 @@ export default {
       },
       update(data) {
         /* istanbul ignore next */
-        return data.validators
+        return data.validators || []
```

Comment:
shouldn't the API always return this, instead of adding conditionals to all our queries?